docs/user/04-metrics.md

```diff
@@ -1,3 +1,395 @@
+
+The goal of the Telemetry Module is to support you in collecting all relevant metrics of workload in a Kyma cluster and ship them to a backend for further analysis. Hereby, relevant Kyma modules like Istio or Serverless will contribute instantly and typical enrichment of the data is happening. Multiple [vendors for OTLP-based backends](https://opentelemetry.io/ecosystem/vendors/) are available.
+
+## Prerequisites
+
+A component from which you want to collect metrics data, needs to expose (or instrument) the metrics first. Typically, it instruments specific metrics for the used language runtime (like nodeJS) and custom metrics specific to the business logic. Also, the exposure can be in different formats, usually the pull-based prometheus format or the [push-based OTLP format](https://opentelemetry.io/docs/specs/otlp/).
+
+To do the instrumentation you usually leverage an SDK, namely the [prometheus-client libraries](https://prometheus.io/docs/instrumenting/clientlibs/) or the [Open Telemetry SDKs](https://opentelemetry.io/docs/instrumentation/). Both libraries provide extensions to activate language specific auto-instrumentation like for nodeJS and an API to implement custom instrumentation.
+
+## Architecture
+
+The Telemetry module provides an in-cluster central Deployment of an [OTel Collector](https://opentelemetry.io/docs/collector/) acting as a gateway. The gateway exposes endpoints for the [OTLP protocol](https://opentelemetry.io/docs/specs/otlp/) for GRPC and HTTP-based communication using the dedicated `telemetry-otlp-metrics` service, where all Kyma components and users' applications should send the metrics data to.
+
+Optional, it provides a DaemonSet of an [OTel Collector](https://opentelemetry.io/docs/collector/) acting as an agent. That agent will scrape metrics of workload in the [prometheus pull-based format](https://prometheus.io/docs/instrumenting/exposition_formats) and can provide runtime specific metrics for workload.
+
+![Architecture](./assets/metrics-arch.drawio.svg)
+
+1. An application exposing metrics in OTLP, pushes metrics to the central metric gateway service 
+2. An application exposing metrics in prometheus protocol, activates the agent to scrape the metrics via aan annotation based configuration
+3. The agent can be activated to scrape metrics of each istio sidecar additionally
+4. The agent convertes and pushes all scraped metric data to the gateway in OTLP
+5. The gateway enrichs all received data with typical metadata of the source by communicating with the K8S APIServer. Furthermore, it will filter data according to the pipeline configuration.
+6. With the `MetricPipeline` resource, the metric gateway is configured with a target backend.
+1. The backend can run in-cluster.
+1. The backend can also run out-cluster, if authentication has been set up.
+1. The metric data can be consumed using the backend system.
```

Comment:
Why "can" ? Is this optional? What else do you do with the metric data?

Could we try:
```suggestion
1. The metric data is consumed using the backend system.
```